app/decorators/authorship_decorator.rb

```diff
@@ -30,6 +30,23 @@ def open_access_status_icon
     end
   end
 
+  def open_access_status_icon_alt_text
+    case open_access_status_icon
+    when 'unlock-alt'
+      'known open access version'
+    when 'lock'
+      'Open access obligations waived'
+    when 'hourglass-half'
+      'Upload to ScholarSphere pending'
+    when 'exclamation-circle'
+      'Scholarsphere upload failed. Please try again'
+    when 'circle-o-notch'
+      'not be subject to the open access policy until published'
```

Comment:
I think it would be good to mention that the publication is "In Press" here.  Maybe just add "publication is in press and will..." to the beginning of this?